tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/worker/WaitActionsNegativeTest.java

```diff
@@ -79,7 +79,9 @@ public void testNegativeWorkerActions() {
         BAssertUtil.validateError(resultNegative, index++,
                 "missing non-defaultable required record field 'f2'", 86, 25);
         BAssertUtil.validateError(resultNegative, index++,
-                "invalid literal for type 'sealedRec'", 87, 26);
+                                  "incompatible types: expected 'sealedRec', found 'record {| other id; other name; " +
+                                          "other status; |}",
```

Comment:
Not related to this, but Saying "other" is confusing here.  We need to come up with a better name for `other`.  One option is to make it special. like, `<unknown_type>` Shall we create an issue?

Reply:
Created #19876 to track this.